option cc_generic_services = true;

message EchoRequest {
	required string text = 1;
	optional int32 times = 2;
}

message EchoResponse {
	required string text = 1;
	optional bool result = 2;
}

service EchoService {
	rpc echo(EchoRequest) returns(EchoResponse);
}
